.github/workflows/erlang.yml

```diff
@@ -0,0 +1,33 @@
+name: Erlang
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  ci:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: gleam-lang/setup-erlang@v1.1.0
```

Comment:
It doesn't work if you remove these two lines?

Reply:
Nope: 
```/usr/bin/ld: cannot find -lerl_interface
collect2: error: ld returned 1 exit status```